src/ctia/auth/jwt.clj

```diff
@@ -186,19 +186,19 @@
       limit-fn)))
 
 (defn unlimited?
-  [unlimited-properties jwt]
-  (let [client-id (get jwt (iroh-claim "oauth/client/id"))
+  [unlimited-properties get-in-config jwt]
```

Comment:
Added as 2nd argument because `unlimited?` is used with `partial` below.

Reply:
yep, that's also one of the reason I preferred to put the service-context as first argument and not the last.

And also to make it easier to support functions with variable number of args.
Fortunately I don't think we have much of those in the code.

Reply:
I see. This isn't a context argument though, it's a service dependency. Is this a reasonable place for it?

Reply:
Merging to keep things moving, but still interested in your opinion here.